README.md

```diff
@@ -16,6 +16,15 @@ def deps do
 end
 ```
 
+By default **you don't need Rust installed** because the lib will try to download
+a precompiled NIF file. In case you want to force compilation set the
+`HTML5EVER_BUILD` environment variable to `true` or `1`. Alternatively you can also set the
+application env `:skip_compilation?` to `false` in order to force the build:
+
+```elixir
+config :html5ever, Html5ever.Native, skip_compilation?: false
```

Comment:
What if we use an option such as `build_from_source: true` to mirror the `_BUILD` env var?

Reply:
@josevalim makes sense! I will do that outside the scope of the NIF module because I would have to change Rustler for that.